api/tests/unit/domain/usecases/import-schooling-registrations-from-siecle_test.js

```diff
@@ -136,34 +148,17 @@ describe('Unit | UseCase | import-schooling-registrations-from-siecle', () => {
       });
 
       it('should throw a FileValidationError', async () => {
-        // when
-        const result = await catchErr(importSchoolingRegistrationsFromSIECLEFormat)({ organizationId, buffer, format, schoolingRegistrationsXmlService: schoolingRegistrationsXmlServiceStub, schoolingRegistrationRepository: schoolingRegistrationRepositoryStub });
-
-        // then
-        expect(result).to.be.instanceOf(FileValidationError);
-        expect(result.message).to.equal('Format de fichier non valide.');
-      });
-    });
-
-    context('because organization UAI imported is not the same as organization UAI registered', () => {
-      beforeEach(async () => {
         // given
-        const extractedStudentsInformations = [{ nationalStudentId: '12345678' }];
-        schoolingRegistrationsXmlServiceStub.extractSchoolingRegistrationsInformationFromSIECLE
-          .returns({ UAIFromSIECLE: organizationUAI, resultFromExtraction: extractedStudentsInformations });
-        schoolingRegistrationRepositoryStub.findByOrganizationId.resolves();
-        schoolingRegistrationRepositoryStub.addOrUpdateOrganizationSchoolingRegistrations.throws(new SameNationalStudentIdInOrganizationError());
-        organizationRepositoryStub.get.withArgs(organizationId).resolves({ externalId: 'otherOrganizationUAI' });
-
-      });
+        const filePath = __dirname + '/wrong-file.txt';
+        payload = { path: filePath } ;
 
-      it('should throw a FileValidationError', async () => {
+        await fs.writeFile(filePath, '');
         // when
-        result = await catchErr(importSchoolingRegistrationsFromSIECLEFormat)({ organizationId, buffer, format, organizationRepository: organizationRepositoryStub, schoolingRegistrationsXmlService: schoolingRegistrationsXmlServiceStub, schoolingRegistrationRepository: schoolingRegistrationRepositoryStub });
+        const result = await catchErr(importSchoolingRegistrationsFromSIECLEFormat)({ organizationId, payload, format, schoolingRegistrationsXmlService: schoolingRegistrationsXmlServiceStub, schoolingRegistrationRepository: schoolingRegistrationRepositoryStub });
 
         // then
         expect(result).to.be.instanceOf(FileValidationError);
-        expect(result.message).to.equal('Aucun étudiant n’a été importé. L’import n’est pas possible car l’UAI du fichier SIECLE ne correspond pas à celui de votre établissement. En cas de difficulté, contactez support.pix.fr.');
+        expect(result.message).to.equal('Format de fichier non valide.');
```

Comment:
Au début je pensais que c'était dû au fait que les données n'étaient pas valides, et donc je me demande si c'était la même chose que 'because the file is not valid', mais en fait c'est que les données sont OK mais que le BDD les rejette
```suggestion
context('because information save has failed in database'', () => {
```